source/js/components/join/join.scss

```diff
@@ -3,7 +3,121 @@
 }
 
 .join-us {
-  &:not(#join-us) .close-button {
+  .join-page-title {
     display: none;
   }
+
+  .join-graphic {
+    display: none;
+  }
+
+  .signup-success .join-graphic {
+    display: block;
+  }
+
+  .join-btn {
+    width: 100%;
+  }
+
+  &.half-width {
+    background: $off-white;
+
+    .join-graphic {
+      @include make-col(12);
+
+      margin-bottom: 3em;
+
+      img {
+        max-width: 5em;
+      }
+    }
+  }
+
+  &.full-width {
+    background: $off-white;
+
+    .join-graphic {
+      @include make-col(12);
```

Comment:
Yeah, they're what Bootstrap is doing under the hood. But because this component has different variations, I needed to include this in the stylesheet instead of inline.